LICENSE

```diff
@@ -401,6 +401,33 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The views and conclusions contained in the software and documentation are those of the authors and
 should not be interpreted as representing official policies, either expressed or implied, of Dmitry Vyukov.
 
+
+This product bundles 'concurrentqueue' which is available a Boost Software License.
```

Comment:
The included header file indicates it is a Simplified/2-Clause BSD.  Is there something else that suggests this is BSL?